lib/u3d/commands_generator.rb

```diff
@@ -148,6 +148,26 @@ def run
         end
       end
 
+      command :dependencies do |c|
+        c.syntax = 'u3d dependencies [-p | --package_manager <package manager>]'
+        c.summary = 'Installs Unity dependencies. [Linux only]'
+        c.description = %(
+          #{c.summary}
+
+          Regarding the package manager: if none if specified, u3d will try to assume it by seeing which one.s is.are installed.
+          Only apt-get and yum are tested right now.
+
+          If several package manager are installed on your machine simultaneously, we recommend that you specify it.
+
+          Regarding the dependencies themselves: only dependencies for the editor are installed. WebGL, Android and Tizen require others that you will have to install manually.
+          More on that: https://forum.unity3d.com/threads/unity-on-linux-release-notes-and-known-issues.350256/
+                  )
+        c.option '-p', '--package_manager STRING', String, 'Specify which package manager to use (apt-get, yum...)'
```

Comment:
do you really want to use it with `-p yum` ?

Reply:
Not a great option, but it was the least bad I could think of. Any suggestion?